### ISSUE TITLE
💡 Edge Add-ons active installation count

### ISSUE THREAD

**Issue by johannchopin** [badges/shields#8688]
:clipboard: **Description**

Add a new badge for the amount of active installation of an **Edge Add-ons** just like the one for the **Chrome Web Store** `/chrome-web-store/users/:storeid`:
![image](https://user-images.githubusercontent.com/31794680/144748664-68e662b9-b711-4ffe-a7a1-f902e75e4570.png)

:link: **Data**

Microsoft seems to have opened a new REST API for the Add-ons. You can have a lot of infos about qn qdd-on by calling:

```
https://microsoftedge.microsoft.com/addons/getproductdetailsbycrxid/{addonsId}
```

Example with [dark-reader](https://microsoftedge.microsoft.com/addons/detail/dark-reader/ifoakfbpdcdoeenechcleahebpibofpc):

https://microsoftedge.microsoft.com/addons/getproductdetailsbycrxid/ifoakfbpdcdoeenechcleahebpibofpc

The result is a JSON object with the property `activeInstallCount`.

:microphone: **Motivation**

It's really important for browser extensions to be easily able to show in a README how much users they already have. It's possible for Chrome and firefox but not edge since there was no API. Now that one is available, it would be nice to implement it. I could try to implement this new badge by my own since it's simply a copy of the **Chrome Web Store** one.

Thanks again for your work on this project 🙏

**Comment by chris48s** [comment]
Is there any documentation for this API?

**Comment by johannchopin** [comment]
They don’t mention it in the official edge Add-ons API https://learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api. However it seems consistent and can be easily tested what do you think?

**Comment by chris48s** [comment]
OK thanks. The reason we say

> Badges should not obtain data from undocumented or reverse-engineered API endpoints.

in [our guidelines](https://github.com/badges/shields/blob/master/CONTRIBUTING.md#badge-guidelines)

is because we had some situations where a contributor worked out that a project had an API by inspecting the frontend code for their website or whatever, added a badge that calls it, and then we ended up relying on an API that wasn't really intended for public use. This then either annoyed the maintainers of the company/project behind it or we had no reasonable expectation of stability/compatibility over time (because devs maintaining the the API were making the assumption the only thing consuming it was their own frontend code) and stuff kept breaking.

That's pretty much the situation we've got here so I am going to close this issue, but we could revisit in future if there is a reliable public source for the data. It looks like this badge suggestion meets all our other criteria.
